index.js

```diff
@@ -113,7 +117,7 @@ Redular.prototype.scheduleEvent = function(name, date, global, data){
   var eventId = shortId.generate();
 
   if(global){
```

Comment:
while we're fixing formatting, can you also update these spaceless-if statements maybe run prettier